blueprints/automation/disable_vacuum_camera_update_when_docked.yaml

```diff
@@ -30,7 +30,7 @@ action:
   - condition: trigger
     id: 0
   - service: |
-      {% if trigger.to_state.state in ["unavailable", "unknown", "docked"] %}
+      {% if trigger.to_state.state in ["unavailable", "unknown", "docked", "idle", "paused"] %}
```

Comment:
In my case "idle" state is charging
![image](https://github.com/user-attachments/assets/de60558f-b16e-4e6f-bd75-d7832c3038bb)

Reply:
@mival what do you mean by that? I think that's expected, e.g. the vacuum is idle but it may be charging (or it may not). I think the camera should be disabled when in that state anyhow.

Reply:
@dahlmo in my case (Roborock S5) when vacuum cleaner is docked the state is `charging`. So in my case I also need disable camera while state is `charging`

Reply:
Thanks, I updated the PR accordingly now.